audio/meson.build

```diff
@@ -18,7 +18,7 @@ foreach m : [
   ['sdl', sdl, files('sdlaudio.c')],
   ['jack', jack, files('jackaudio.c')],
   ['sndio', sndio, files('sndioaudio.c')],
-  ['pw', pipewire, files('pipewire.c')],
+  ['pw', pipewire, files('pwaudio.c')],
```

Comment:
You should squash this with the previous commit, which would actually break mainline without this commit to fix it.